### PR TITLE
Auth react admin

### PR DESCRIPTION
**Pull request by Fx-Colliot**

### CODE REVIEW

**Review by Thoms-C — DISMISSED** [review]
ok!

File: server.js [hunk 1]
```diff
@@ -10,6 +10,12 @@ app.use(cors());
 /* --------------------------------------------------------------------- Routes */
 
 app.use("/auth", require("./routes/auth"));
+
+app.use("/", require("./routes/projets"));
+app.use("/", require("./routes/logos"));
+app.use("/", require("./routes/docFavart"));
+app.use("/", require("./routes/docExt"));
```
Comment (LoicBrassart) [inline server.js]: Nope, that's not how you work with routers, you should have something like this:
```
app.use("/projets", require("./routes/projets"));
￼app.use("/logos", require("./routes/logos"));
￼app.use("/resources", require("./routes/docFavart"));
￼app.use("/resourcesext", require("./routes/docExt"));
```

File: package.json [hunk 2]
```diff
@@ -4,12 +4,13 @@
...
   "main": "index.js",
   "scripts": {
-    "start": "nodemon index.js"
+    "start": "nodemon server.js"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "axios": "^0.21.1",
```
Comment (LoicBrassart) [inline package.json]: Why do you want to install Axios on your backend ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some changes to do

**Review by Thoms-C — APPROVED** [review]
requests for route done

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**